waspc/e2e-test/test-outputs/waspMigrate-golden/files.manifest

```diff
@@ -3,7 +3,6 @@ waspMigrate/.wasp/out/.waspchecksums
 waspMigrate/.wasp/out/Dockerfile
 waspMigrate/.wasp/out/db/migrations/migration_lock.toml
 waspMigrate/.wasp/out/db/migrations/no-date-foo/migration.sql
-waspMigrate/.wasp/out/db/package.json
```

Comment:
@sodic This must mean we dropped having `package.json` in the `db` folder sometime between last time e2e tests were generated and this PR.

Reply:
Ok we concluded this was caused by a prisma package.json bug